data/publiccloud/azure_more_cli/azure_vn.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -u
 set -o pipefail
-# This bash source is for reporting
+# This bash source is for reporting which is common
+# for all azure_more_cli testing
 # shellcheck disable=2046
 # shellcheck disable=1091
 . $(dirname "${BASH_SOURCE[0]}")/azure_lib_fn.sh
```

Comment:
Please follow shellcheck's suggestion which is:

```
In data/publiccloud/azure_more_cli/azure_vn.sh line 4:
. $(dirname "${BASH_SOURCE[0]}")/azure_lib_fn.sh
  ^----------------------------^ SC2046 (warning): Quote this to prevent word splitting.
  ^-- SC1091 (info): Not following: ./azure_lib_fn.sh was not specified as input (see shellcheck -x).
```

```suggestion
. "$(dirname "${BASH_SOURCE[0]}")"/azure_lib_fn.sh
```

Reply:
resolved for all test shell scripts